src/features/components/quill/layout-overlay/live-chat-button/index.tsx

```diff
@@ -115,7 +116,13 @@ const LiveChatButton = () => {
             {is_livechat_interactive ? (
                 <FloatingButton
                     className="gtm-deriv-livechat"
-                    onClick={() => LC_API.current.open_chat_window()}
+                    onClick={() => {
```

Comment:
Can we have function in utility and use it here. Would be good for re-usabilty

Reply:
it's possible. I moved that function to common folder

Reply:
yes, it's possible. I add convert that function to ulility one and move to /common folder